### PR TITLE
ci: Disable intermittently failing MFA e2e tests (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
MFA e2e tests keep intermittently failing, creating distractions

Capture 2023-09-05 at 09 36 45@2x

Followup story to fix them: https://linear.app/n8n/issue/N8N-6896

### CODE REVIEW

File: cypress/e2e/27-two-factor-authentication.cy.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ describe('Two-factor authentication', () => {
 		});
 	});
 
-	it('Should be able to login with MFA token', () => {
+	it.skip('Should be able to login with MFA token', () => {
```
Comment (mutdmour) [inline cypress/e2e/27-two-factor-authentication.cy.ts:40]: can we try to fix them instead of disabling?

**Review by mutdmour — CHANGES_REQUESTED** [review]

File: cypress/e2e/27-two-factor-authentication.cy.ts (hunk 1 above)
Comment (ivov) [reply cypress/e2e/27-two-factor-authentication.cy.ts:40]: The description has the followup story for Ricardo, already discussed with him.

**Review by mutdmour — APPROVED** [review]

File: cypress/e2e/27-two-factor-authentication.cy.ts (hunk 1 above)
Comment (mutdmour) [reply cypress/e2e/27-two-factor-authentication.cy.ts:40]: @RicardoE105 please follow up on this asap

**Review by flipswitchingmonkey — APPROVED** [review]

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.8.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.8.1)

**Pull request merged**